salt/modules/file.py

```diff
@@ -3841,10 +3842,23 @@ def set_selinux_context(path,
 
         salt '*' file.set_selinux_context path <user> <role> <type> <range>
         salt '*' file.set_selinux_context /etc/yum.repos.d/epel.repo system_u object_r system_conf_t s0
+
+    .. versionchanged:: Neon
+
+        Added persist option
```

Comment:
Sorry for this nitpick, but we try to put our versionchanged/versionadded notices at the top of the docstring rather than the bottom. Could you move this up?

Reply:
No problem! Done!